packages/flutter/lib/src/material/about.dart

```diff
@@ -456,9 +466,12 @@ class _LicensePageState extends State<LicensePage> {
     assert(debugCheckHasMaterialLocalizations(context));
     final String name = widget.applicationName ?? _defaultApplicationName(context);
     final String version = widget.applicationVersion ?? _defaultApplicationVersion(context);
+    final Widget icon = widget.applicationIcon ?? _defaultApplicationIcon(context);
     final MaterialLocalizations localizations = MaterialLocalizations.of(context);
     final List<Widget> contents = <Widget>[
       Text(name, style: Theme.of(context).textTheme.headline, textAlign: TextAlign.center),
+      if (icon != null)
+        IconTheme(data: const IconThemeData(size: 48.0), child: icon),
```

Comment:
The API docs say: `Typically this will be an [ImageIcon] widget. It should honor the [IconTheme]'s [IconThemeData.size].` 

Doesn't this line force it to be of size 48.0?

Reply:
It is forced to be size 48.0 now. Thanks for your comment, I will fix this line and existing `AboutDialog`.
https://github.com/flutter/flutter/blob/04bd314dcedd2c290ad195d3f6693651c94ad095/packages/flutter/lib/src/material/about.dart#L274

Reply:
I fixed icon size forcing by following lines.

* https://github.com/flutter/flutter/pull/34906/files#diff-0ff29c7d3513e662be116d2052bd3f97R488
* https://github.com/flutter/flutter/pull/34906/files#diff-0ff29c7d3513e662be116d2052bd3f97R279